airbyte_cdk/sources/declarative/incremental/concurrent_partition_cursor.py

```diff
@@ -245,7 +242,7 @@ def _ensure_partition_limit(self) -> None:
                             partition_key
                         )  # Remove the oldest partition
                         logger.warning(
-                            f"The maximum number of partitions has been reached. Dropping the oldest finished partition: {oldest_partition}. Over limit: {self._over_limit}."
+                            f"The maximum number of partitions has been reached. Dropping the oldest finished partition: {oldest_partition}. Over limit: {self._number_of_partitions}."
```

Comment:
Should this log be updated? It seems like we print the number of partitions and not the number over the limit. The same comment applies to the log below